run-tests.sh

```diff
@@ -36,7 +36,7 @@ data_dois() {
 }
 
 data_json() {
-    find . -name "*.json" -exec ./scripts/clean_json_file.py --check {} \;
+    find . -name "*.json" -exec ./scripts/clean_json_file.py --check {} \+
```

Comment:
This will pass all the files in a single command, isn't it? It is better than the previous version. At the same time, wouldn't it be even better if the script accepted a directory, and it would go through all the json files in that directory?

Reply:
That could be another option. I went for accepting multiple file names instead, just like `jsonlint` (or `shellcheck`) are doing...